content/admin.py

```diff
@@ -55,6 +55,13 @@ class VolunteerAboutAdmin(admin.ModelAdmin):
 class CustomPageAdmin(admin.ModelAdmin):
     list_display = ('category', 'order','title')
 
+class DonateInfoAdmin(admin.ModelAdmin):
+    fields = [ 'order', 'title', 'img', 'text']
+    list_display = ('order','title')
```

Comment:
I did this to give an order for fields in admin when entering info. Also, To list certain fields 
![adminlist](https://cloud.githubusercontent.com/assets/10239547/15266461/6c6c70ac-19e0-11e6-8f1c-0f35fe809c82.png)